index.html

```diff
@@ -20,6 +20,28 @@
 table { border-spacing: 1px 4px; }
 body { font-family: Arial, sans-serif; }
 
+body{ background-image: linear-gradient(90deg, orange 60%, cyan); } /* Added a background colour */
+
+.intro{                                 /*for making the top heading in center*/
```

Comment:
Hmmm, I'm not sure it would look better with everything centered– human reading starts from the left, after all. I suppose it might look nicer if we moved the key and heading towards the center, but kept them left-aligned. I'm not sure this looks good when the key isn't also centered.